app/uk/gov/hmrc/exports/services/email/EmailSender.scala

```diff
@@ -37,7 +37,10 @@ class EmailSender @Inject()(
     obtainEori(mrn).flatMap {
       case Some(eori) =>
         obtainEmailAddress(eori).flatMap {
-          case Some(verifiedEmailAddress) => sendEmail(mrn, eori, verifiedEmailAddress)
+          case Some(Email(email, true)) => sendEmail(mrn, eori, Email(email, deliverable = true))
```

Comment:
You could simplify this line with `Some(email) => sendEmail(mrn, eori, email)`. At present you are deconstructing the `Email` case class just to re-construct it again :wink: